cdc/redo/manager.go

```diff
@@ -566,18 +588,23 @@ func (m *ManagerImpl) bgUpdateLog(
 		}
 	}
 
+	overseerTimer := time.NewTicker(time.Second * 5)
```

Comment:
It is better to name it ticker rather than timer and use a const rather than a magic number `time.Second * 5`.